src/CONST.ts

```diff
@@ -3210,6 +3214,29 @@ const CONST = {
         REPORT: 'REPORT',
     },
 
+    THUMBNAIL_IMAGE: {
+        SMALL_SCREEN: {
+            SIZE: 250,
+        },
+        WIDE_SCREEN: {
+            SIZE: 350,
+        },
+        NAN_ASPECT_RATIO: 1.5,
+    },
+
+    VIDEO_PLAYER: {
+        POPOVER_Y_OFFSET: -30,
+        PLAYBACK_SPEEDS: [0.25, 0.5, 1, 1.5, 2],
```

Comment:
Turns out that `PLAYBACK_SPEEDS` required some additional granularity which led to this issue:
- https://github.com/Expensify/App/issues/47205

We fixed the issue by adding a wider range of `PLAYBACK_SPEEDS`, more details in the ^ issue / [proposal](https://github.com/Expensify/App/issues/47205#issuecomment-2282923395).